package.json

```diff
@@ -19,11 +19,11 @@
   "author": "SubQuery Network",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@subql/types": "latest",
-    "@subql/testing": "latest",
-    "@subql/node": "latest",
-    "@subql/cli": "latest",
     "@polkadot/api": "^10",
+    "@subql/cli": "^4.2.1",
```

Comment:
Was there and issue to use "latest" here?

Reply:
On another note, I would say using an actual version could be a better choice than "latest". So you may want to use the version uniformly for all of "subql" dependencies.

Reply:
Yes, had some issues with the package manager Yarn

Indeed, I already had issues related to that, that time only with `@subql/cli`